runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -191,7 +179,7 @@ public void init()
 		}
 
 		Widget bankContainer = client.getWidget(WidgetInfo.BANK_CONTAINER);
```

Comment:
You can probably just get the widgetIds directly from BANK_CONTAINER widget in the BankSearch when calling the method (getting widget has almost no overhead) instead of setting them here

Reply:
Rebased and moved.